sa.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-07/schema
+title: GA4GH-SA-Definitions
+type: object
+definitions:
+  SequenceFeature:
+    properties:
+      id:
+        type: string
+      location:
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/main/schema/vrs.yaml#definitions/Location"
+      seqOntologyRef:
+        type: string
+    required:
+    - id
+    - location
+    - seqOntologyRef
+    oneOf:
+    - "$ref": "#/definitions/Gene"
+    - "$ref": "#/definitions/Transcript"
+    - "$ref": "#/definitions/TranscriptRegion"
+  Gene:
+    properties:
+      transcripts:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "#/definitions/Transcript"
+      strand:
+        type: string
+        enum:
+        - "+"
+        - "-"
+      referenceGenome:
+        type: string
+    required:
+    - strand
+    additionalProperties: true
+  Transcript:
+    properties:
+      coding:
+        type: boolean
```

Comment:
Would this apply to transcripts undergoing NMD? Is it a flag to say it has a protein product? What about active biomolecular products like lncRNA or miRNA?

Reply:
@andrewyatz I'm still trying to think of a good way to represent sequence features that map to multiple types of sequence. I think we may want to explore Bob's idea of MolecularState that he presented at the last connect meeting.